docs/strategies/retry.md

```diff
@@ -108,6 +108,55 @@ new ResiliencePipelineBuilder<HttpResponseMessage>().AddRetry(optionsExtractDela
 | `OnRetry`          | `null`                                                                     | Action executed when retry occurs.                                                       |
 | `MaxDelay`         | `null`                                                                     | Caps the calculated retry delay to a specified maximum duration.                         |
 
+## Calculation of the next delay
+
+If the `ShouldHandle` predicate returns `true` and the next attempt number is not greater than `MaxRetryAttempts` then the retry strategy calculates the next delay.
+
+There are many properties that are somehow contribute to this calculation:
+
+- `Delay`
+- `DelayGenerator`
+- `MaxDelay`
+- `UseJitter`
+- `BackoffType`
+
+> [!IMPORTANT]
+> The below described algorithm is an implementation detail. It might change in the future without further notice.
+>
+> Also please bear in mind that some details are not mentioned here to keep the algorithm description short and terse.
+
+The `BackoffType` property's data type is a [`DelayBackOffType`](xref:Polly.DelayBackOffType) enum. This controls primarily how the calculation is done.
+
+### Constant
+
+One might think that here we only put into the account the `Delay` property. In reality all above listed properties are used.
+
+Step 1: Calculating the base delay:
+
+- If `UseJitter` is set to `false` and `Delay` is specified then `Delay` will be used.
+- If `UseJitter` is set to `true` and `Delay` is specified then a random value is added to the `Delay`.
+  - The random value is between [-25% of `Delay`, +25% of `Delay`].
```

Comment:
Looks weird to me to use the word between but then use range notation in the middle of prose.

Reply:
```suggestion
  - The random value is between -25% of `Delay` and +25% of `Delay`.
```
Is it better?